policy-engine/src/metrics.rs

```diff
@@ -31,3 +31,37 @@ pub(crate) fn serve(
         .map(|content| HttpResponse::Ok().body(content));
     Box::new(resp)
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::metrics;
+    use actix_web::test::TestRequest;
+    use failure::Fallible;
+    use tokio::runtime::current_thread::Runtime;
+
+    fn common_init() -> Fallible<Runtime> {
```

Comment:
could `common_init` be shared between these? maybe in a `tests/lib.rs` file?

Reply:
They are bit-by-bit identical. The problem is that they live in two separate top-level **binary crates**. The only way that comes to my mind is via a third crate library, but that sounds a bit gross.

Reply:
I think it's fine having them in the *common* crate.